04/app.js

```diff
@@ -9,14 +10,49 @@ class App extends React.Component {
         filesList: [],
     }
 
+    readFile = e => {
+        for (let i = 0; i < e.target.files.length; i++) {
+            const file = e.target.files[i];
+
+            let infoFile = {};
+            if (file && file.type.includes('text')) {
+                const reader = new FileReader();
+                reader.onload = function (readerEvent) {
+                    const content = readerEvent.target.result;
+
+                    infoFile = {
+                        id: uuid(),
+                        name: file.name,
+                        size: file.size,
+                        content: content,
+                    }
+                };
+                reader.readAsText(file, 'UTF-8');
+
+                setTimeout(() => { this.addFile(infoFile) }, 500);
```

Comment:
Dlaczego wykonujesz to z opóźnieniem? Raczej `this.addFile` powinno być wew. `reder.onload()`
Aby to zadziałało musisz zrobić funkcje strzałkową w `reader.onload ` czyli `reader.onload = (readerEvent) => {`